src/render/ps2/SDL_render_ps2.c

```diff
@@ -103,6 +103,27 @@ static int PixelFormatToPS2PSM(Uint32 format)
     }
 }
 
+static gs_rgbaq float_color_to_RGBAQ(const SDL_FColor *color)
+{
+    uint8_t colorR = (uint8_t)(color->r * 127.0f);
```

Comment:
I was originally wrongly doing `>> 1`, however after taking a look at the documentation it should be `255.0f` in all the channel

Reply:
Okay, I'll fix that.